records_mover/records/cli.py

```diff
@@ -101,6 +99,7 @@ def main() -> None:
     if func is None:
         parser.print_help()
     else:
-        log_levels.set_levels(name)
-        cli_logging.basic_config()
+        from records_mover import Session
+        session = Session()
+        session.set_stream_logging()
```

Comment:
Creating a throwaway session object here is kind of awkward.  I suppose we could create set_stream_logging() as an independent function you can call outside of a session (in addition to being a method on the session?)/

Reply:
Fwiw, boto3 seems to have the logging helper outside of the session object, in contrast to botocore which has it in the session object (but doesn't actually make use of `self` in any meaningful way and so could just as well live as a top-level function like in boto3)

Reply:
I'm gonna propose we do both here.